packages/contracts/test/voting/token-voting.ts

```diff
@@ -705,6 +715,14 @@ describe('TokenVoting', function () {
         expect(proposal.tally.abstain).to.equal(10);
       });
 
+      it('reverts on voting None', async () => {
```

Comment:
The same here as above.

Reply:
Added in 0fed43072dcd874cd9b4d43a70e0a28bc95a39e0